lib/miq_automation_engine/models/miq_ae_class.rb

```diff
@@ -12,7 +12,8 @@ class MiqAeClass < ApplicationRecord
 
   validates_presence_of   :name, :namespace_id
   validates_uniqueness_of :name, :case_sensitive => false, :scope => :namespace_id
-  validates_format_of     :name, :with => /\A[A-Za-z0-9_.-]+\z/i
+  validates_format_of     :name, :with    => /\A[A-Za-z0-9_.-]+\z/i,
```

Comment:
Can't we simplify this regular expression like this? Similar for others below.

`/\A[\w.-]+\z/i`